lib/core/public/configure.js

```diff
@@ -8,6 +8,21 @@ function configureChecksRulesAndBranding(spec) {
 		throw new Error('No audit configured');
 	}
 
+	if (spec.ver) {
+		let [major, minor, patch] = spec.ver.split('.').map(Number);
+		let [axeMajor, axeMinor, axePatch] = axe.version.split('.').map(Number);
+
+		if (
+			major !== axeMajor ||
+			axeMinor < minor ||
+			(axeMinor === minor && axePatch < patch)
```

Comment:
Should this be `axePatch <= patch`?

Reply:
we shouldn't error if the patches are equal

Reply:
Good point. My bad.